src/rhsm/profile.py

```diff
@@ -36,8 +36,16 @@
 except ImportError:
     yum = None
 
+try:
+    import zypp_plugin # noqa: F401
```

Comment:
It's better but still not perfect ;-)
```
src/rhsm/profile.py:40:23: E261 at least two spaces before inline comment
```
You can test style of your file using: `flake8 src/rhsm/profile.py` before pushing to server.

Reply:
ah. sorry for that. Thanks for you help. Fixed it now.